client/package.json

```diff
@@ -72,6 +72,7 @@
     "@testing-library/jest-dom": "^5.0.2",
     "@testing-library/react": "^16.0.1",
     "@types/js-cookie": "^3.0.6",
+    "@types/node": "^22.9.0",
```

Comment:
Is this being used?

Reply:
I must have inadvertently updated that. I'll remove that.